vendor/github.com/elastic/go-libaudit/CHANGELOG.md

```diff
@@ -0,0 +1,64 @@
+# Change Log
```

Comment:
I can, but how do I accomplish this with `govendor`? I end up with shadow deps in module/audit/vendor for `github.com/pkg/errors` and `gopkg.in/yaml.v2` because they are used by `github.com/elastic/go-libaudit/...`.

```
cd modules/audit
govendor init
govendor fetch github.com/elastic/go-libaudit/...@v0.0.3
tree -L 2 vendor/
vendor/
├── github.com
│   ├── elastic
│   └── pkg
├── gopkg.in
│   └── yaml.v2
└── vendor.json
```